### PR TITLE
Isce3 rtc

### PR DESCRIPTION
**Pull request by Alex-Lewandowski**
Add isce3_rtc env that installs OPERA's s1-reader and rtc packages

### CODE REVIEW

**Review by ruikawahara — APPROVED** [review]
Reviewed changes for ISCE3 - confirmed the addition of `.yml` file, helper script, and proper adjustment for env creation notebook.

File: Scripts/isce3_rtc.sh
Comment (dgpalmieri) [inline Scripts/isce3_rtc.sh:1]: We should merge in the changes to this script from the binder branch, I fixed a couple bugs there that are still present here.

**Review by dgpalmieri — CHANGES_REQUESTED** [review]
Need to make a couple changes to the `isce3_rtc.sh` script that I made in the [isce3_rtc binder](https://github.com/ASFOpenSARlab/opensarlab-notebooks/blob/binder_isce3_rtc/binder/postBuild)

**Comment by dgpalmieri** [comment]
These changes from https://github.com/OPERA-Cal-Val/calval-RTC/pull/6

**Comment by dgpalmieri** [comment]
I've verified that the env does build and can successfully run the isce3 notebook

**Comment by Alex-Lewandowski** [comment]
Add isce3_rtc env

**Pull request merged**